### DISCUSSION TITLE
Is it possible to provide more infomation, like '放銃率'?

### DISCUSSION THREAD

**Discussion by ccc1994** [Equim-chan/mjai-reviewer#95] [Q&A]
First, sorry about my poor english. Recently i'm using 'mjai.ekyu.moe' to improve my majang skill, it helps me a lot, thanks for your great work.
But sometimes when my chosen was different with ai, i can only guess the reason, but if more information provided, like '放銃率', maybe i can more sure about it?

**Comment by Equim-chan** [comment]
For Mortal backend, that will require lots of extra works, and it can't be accurate anyways, so it's not planned.
